pkg/logging/logging.go

```diff
@@ -0,0 +1,98 @@
+package logging
+
+import (
+	"fmt"
+
+	"github.com/go-logr/logr"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+/*
+Log logs messages to various levels.
+
+Based on the go-logr package.
+
+The Log type enables various verbosity levels to be logged based on to the --zap-level argument to the operator.
+
+To create an instance of the logger:
+
+    log := logging.Logger("main")
+
+When --zap-level is not provided, the logging defaults to INFO level where log.Warning() and log.Info() are logged,
```

Comment:
Could you make sure lines have the same width than specified in `.editorconfig`?